machine/machine_core/machine.py

```diff
@@ -235,6 +235,10 @@ def allowed_messages(self):
             # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1979182
             allowed.append('audit.*denied.*{ getattr }.*comm="systemctl" name="/".*cockpit_ws_t.*')
 
+        if self.image in ['rhel-8-6', 'centos-8-stream']:
```

Comment:
This is already done in #2681, so I suppose this PR needs to be rebased after that lands.

Reply:
Yeah, I just wanted to run the tests in parallel. This will get sorted out during rebasing.